datamodel/src/main/kotlin/com/hartwig/actin/datamodel/algo/TreatmentMatch.kt

```diff
@@ -10,6 +11,22 @@ data class TreatmentMatch(
     val trialMatches: List<TrialMatch>,
     val standardOfCareMatches: List<AnnotatedTreatmentMatch>?,
     val personalizedDataAnalysis: PersonalizedDataAnalysis?,
-    val survivalPredictionsPerTreatment: Map<String, List<Double>>?,
+    val survivalPredictionsPerTreatment: Map<String, TreatmentEfficacyPrediction>?,
     val maxMolecularTestAge: LocalDate? = null
 )
+
+data class TreatmentEfficacyPrediction(
+    @JsonProperty("survival_probs")
```

Comment:
This is tricky... I don't think we should expose non-standard json formatting downstream. This data class should be clean, but when we read it in from run_prediction we should map it (only there). Hope this makes sense!